modules/abi-oci-resources-services.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="abi-oci-resources-services_{context}"]
+= Creating {oci} infrastructure resources and services
+
+You must create an {oci} environment on your virtual machine (VM) shape. By creating this environment, you can install {product-title} and deploy a cluster on an infrastructure that supports a wide range of cloud options and strong security policies. Having prior knowledge of {oci} components can help you with understanding the concept of {oci} resources and how you can configure them to meet your organizational needs.
+
+The Agent-based installer method for installing an {product-title} cluster on {oci} requires that you manually create {oci} resources and services.
+
+[IMPORTANT]
+====
+To ensure compatibility with {product-title}, you must set `A` as the record type for each DNS record and name records as follows:
+
+* `api.<cluster_name>.<base_domain>`, which targets the `apiVIP` parameter of the API load balancer.
+* `api-int.<cluster_name>.<base_domain>`, which targets the `apiVIP` parameter of the API load balancer.
+* `*.apps.<cluster_name>.<base_domain>`, which targets the `ingressVIP` parameter of the Ingress load balancer.
+
+The `api.+*+` and `api-int.+*+` DNS records relate to control plane machines, so you must ensure that all nodes in your installed {product-title} cluster can access these DNS records.
+====
+
+.Prerequisites
+
+* You configured an {oci} account to host the {product-title} cluster. See link:https://docs.oracle.com/iaas/Content/openshift-on-oci/install-prereq.htm[Prerequisites] in the _Oracle_ documentation.
+
+.Procedure
+
+* Create the required {oci} resources and services. See link:https://docs.oracle.com/iaas/Content/openshift-on-oci/agent-prereq.htm[{oci} Resources Needed for Using the {product-title} Agent-based Installer (Oracle documentation)].
```

Comment:
```suggestion
* Create the required {oci} resources and services. See link:https://docs.oracle.com/iaas/Content/openshift-on-oci/agent-prereq.htm[{oci} Resources Needed for Using the {product-title} Agent-based Installer] in the  _Oracle documentation_.
```

Reply:
No change as meets https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute